Dockerfile.cra

```diff
@@ -7,6 +7,7 @@ ENV PATH /app/node_modules/.bin:$PATH
 COPY package.json ./
 COPY yarn.lock ./
 COPY . ./
+RUN apk add python3 make gcc g++
```

Comment:
Extra things for gyp when building on arm. They are subtely different, the next apps we need to run `export` not `build` so that is the only difference right now. I am not docker savvy enough to know if we could share these. I suppose we could take a build arg for that the command to run is. Though I think this is fine until we see how much these are needed.